testing/tests/src/main/kotlin/CompileTestDriverBase.kt

```diff
@@ -170,6 +170,7 @@ abstract class CompileTestDriverBase private constructor(
             "-opt-in=com.yandex.yatagan.ConditionsApi",
             "-opt-in=com.yandex.yatagan.VariantApi",
             "-P", "plugin:org.jetbrains.kotlin.kapt3:correctErrorTypes=true",
+            "-jvm-target=11",
```

Comment:
Btw, why tests are running on Java 11, but target is 8?

Reply:
Testing framework requires 11+, but yatagan itself is built with 8 to maximize compatibility.